### DISCUSSION TITLE
trim_pores error message using openpnm dev version

### DISCUSSION THREAD

**Discussion by minzhangut** [PMEAL/OpenPNM#2433] [Q&A]
I got an error saying:
`Exception: Your network is clustered. Run h = net.check_network_health() followed by op.topotools.trim(net, pores=h['trim_pores']) to make your network fully connected.
`

I found that there are "isolated_pores" and "disconnected_pores" instead of "trim_pores", so I used `op.topotools.trim(network=pn, pores=h['disconnected_pores'])` instead.

So I think the error message from openpnm should be updated.

**Comment by jgostick** [comment]
This is technically a bug report, so I will convert it.  Thanks @minzhangut

**Reply by ma-sadeghi** [reply]
@minzhangut Do you mind creating a pull request with the fix you mentioned? Would greatly appreciate it.

**Reply by minzhangut** [reply]
@ma-sadeghi I would love to help. However, I am not sure what do you want me to do?

**Reply by ma-sadeghi** [reply]
I'm not sure how familiar you are with git. Here's what you need to do:

- Fork OpenPNM repository by clicking the "Fork" button (top right)
- Using a Git client (like [Fork](https://git-fork.com/)), clone your own fork (not PMEAL's). Your fork URL will be something like: https://github.com/minzhangut/OpenPNM
- Inside the Git client, create a new branch and name it whatever you like
- Using a text editor (or Spyder if that's what you're using), open `OpenPNM/openpnm/algorithms/_generic_transport.py` (this should be coming from your Fork, not your `pip`/`conda` installation) and make the edit you suggested (modify the error message). The error message is roughly in line 255.
- Using the Git client, "commit" (it's a git concept) the change you just made, and "push" (another git concept) it to your OpenPNM fork.
- Open your OpenPNM fork in a browser (probably https://github.com/minzhangut/OpenPNM), go to the "Pull Requests" tab and create a new pull request, set the branch you have made as the "source" branch, and PMEAL/dev as the destination branch.

If these steps make sense to you, then it's all good, otherwise, I can prepare a "Contributing Guide"  document with more details and put it on the main repo so you (and hopefully other potential contributors) can use. Please let me know.

**Comment by mzdreambig** [comment]
Hi @ma-sadeghi , I think I followed your instructions using my another account @mzdreambig. Please let me know what I should do next. Thanks!

**Reply by ma-sadeghi** [reply]
@mzdreambig Thank you! You did great, and I merged your pull request. If you pull the latest code, it should be fixed. Thanks once again!
